nanofi/src/cxx/Plan.cpp

```diff
@@ -162,6 +153,21 @@ bool ExecutionPlan::runNextProcessor(std::function<void(const std::shared_ptr<co
   }
   std::shared_ptr<core::ProcessSession> current_session = std::make_shared<core::ProcessSession>(context);
   process_sessions_.push_back(current_session);
+  if (input_ff != nullptr) {
+    auto content_repo = static_cast<std::shared_ptr<minifi::core::ContentRepository>*>(input_ff->crp);
+    std::shared_ptr<minifi::ResourceClaim> claim = std::make_shared<minifi::ResourceClaim>(input_ff->contentLocation, *content_repo);
+    auto stream = (*content_repo)->read(claim);
```

Comment:
This copies the content of the incoming flow file. 
Naturally this leaves space for improvement, in case the content repos of the current flow and the incoming flow file are the same, the copy is needless. 
Given the size of the change I would prefer to do it later.